pkg/apis/apis/v1alpha1/types.go

```diff
@@ -355,6 +355,12 @@ type APIExportSpec struct {
 	// i.e the consuming users can do whatever the user workspace allows the user to do.
 	// +optional
 	MaximalPermissionPolicy *APIExportPolicy `json:"maximalPermissionPolicy,omitempty"`
+
+	// permissionClaims will allow the APIExport creator to ask for permissions in the given bound workspace
```

Comment:
Global suggestion: frame it as what the system does, not what it will do.

Reply:
Also, AFAIK newlines are not preserved in the API schema docs. Make sure to end all sentences with a period 😄

Reply:
resolved I think.